test/unit/murmurhash3_spec.js

```diff
@@ -0,0 +1,53 @@
+/* globals jasmine, expect, it, describe, MurmurHash3_64 */
+'use strict';
+
+describe('MurmurHash3_64', function() {
+
+  it('instantiates without seed ', function() {
+      var hash = new MurmurHash3_64();
```

Comment:
Indent this line with two spaces instead of four, i.e., remove two spaces. We always indent with two spaces. This also applies to the other occurrences in this file.